### PR TITLE
[11.0][MIG] 11.0 payment transfer

### PR DESCRIPTION
**Pull request by nadiaafa**
Depends on:

- [x] payment - #1197

### CODE REVIEW

**Review by StefanRijnhart — CHANGES_REQUESTED** [review]
Thanks! Please resubmit without the hr commit.

**Comment by nadiaafa** [comment]
@StefanRijnhart changes done, thank you

File: addons/payment_transfer/migrations/11.0.1.0/noupdate_changes.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+  <record id="payment.payment_acquirer_transfer" model="payment.acquirer">
+    <field name="pre_msg"><![CDATA[
+        <p>Transfer information will be provided after choosing the payment aquirer.</p>]]></field>
+    <field name="view_template_id" ref="transfer_form"/>
+    <field name="auto_confirm"/>
+  </record>
+</odoo>
```
Comment (MiquelRForgeFlow) [inline addons/payment_transfer/migrations/11.0.1.0/noupdate_changes.xml]: add new line below

**Review by MiquelRForgeFlow — APPROVED** [review]

File: addons/payment_transfer/migrations/11.0.1.0/noupdate_changes.xml (hunk 1 above)
Comment (nadiaafa) [reply addons/payment_transfer/migrations/11.0.1.0/noupdate_changes.xml]: line added

File: addons/payment_transfer/migrations/11.0.1.0/noupdate_changes.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+  <record id="payment.payment_acquirer_transfer" model="payment.acquirer">
+    <field name="pre_msg"><![CDATA[
+        <p>Transfer information will be provided after choosing the payment aquirer.</p>]]></field>
+    <field name="view_template_id" ref="transfer_form"/>
+    <field name="auto_confirm"/>
```
Comment (StefanRijnhart) [inline addons/payment_transfer/migrations/11.0.1.0/noupdate_changes.xml]: Same as https://github.com/OCA/OpenUpgrade/pull/1197#pullrequestreview-78895240

**Comment by pedrobaeza** [comment]
WIP until `payment` is merged.

File: addons/payment_transfer/migrations/11.0.1.0/noupdate_changes.xml (hunk 2 above)
Comment (nadiaafa) [reply addons/payment_transfer/migrations/11.0.1.0/noupdate_changes.xml]: @StefanRijnhart Thank you, i removed auto_confirm

**Comment by pedrobaeza** [comment]
As @StefanRijnhart's comments have been honored, I merge this.

**Pull request merged**